packages/yoga/src/BottomSheet/web/BottomSheet.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styled from 'styled-components';
+import { bool, node } from 'prop-types';
+import Dialog from '../../Dialog';
+import { StyledDialog } from '../../Dialog/web/Dialog';
+
+const StyledBottomSheet = styled(Dialog)`
+  ${({
+    theme: {
+      yoga: {
+        components: { bottomsheet },
+      },
+    },
+  }) => `
+    align-items: end;
+    z-index: 10;
+
+    ${StyledDialog} {
```

Comment:
Based on my previous suggestion, here instead of this you'd export Dialog's Overlay to use in this file instead